docs/README.md

```diff
@@ -0,0 +1,3 @@
+# About Thunderbird for Android
+
+This document provides an overview of the Thunderbird for Android project.
```

Comment:
It might make sense to add link to the generated docs here, for people browsing the folder.

Reply:
How does this file relate to SUMMARY.md. Do we need both?

Reply:
The SUMMARY.md is used to define the navigation within the docs.

The README.md is emtpy at the moment, but it should be extended to give developers and contributors a quick overview of the docs content. I thought about the same approach as: https://developer.thunderbird.net/